internal/server/worker.go

```diff
@@ -108,12 +123,13 @@ func NewWorker(scopeId string, opt ...Option) *Worker {
 	opts := GetOpts(opt...)
 	worker := &Worker{
 		Worker: &store.Worker{
-			ScopeId:          scopeId,
-			Name:             opts.withName,
-			Description:      opts.withDescription,
-			Address:          opts.withAddress,
-			ReleaseVersion:   opts.withReleaseVersion,
-			OperationalState: opts.withOperationalState,
+			ScopeId:           scopeId,
+			Name:              opts.withName,
+			Description:       opts.withDescription,
+			Address:           opts.withAddress,
+			ReleaseVersion:    opts.withReleaseVersion,
+			OperationalState:  opts.withOperationalState,
+			LocalStorageState: opts.withLocalStorageState,
```

Comment:
We should add a new unit test for the internal/server/worker_test.go file to validate the LocalStorageState. We can add a test case to the `TestWorker_New` method.